workflows/cloudwatch_logs_large_query/README.md

```diff
@@ -0,0 +1,62 @@
+# CloudWatch Logs large query
+
+## Overview
+
+This example shows how to use AWS SDKs to perform a query on CloudWatch logs and get more than the maximum amount of 10,000 logs back.
+
+The CloudWatch Logs API is capped at 10,000 records for requests that [read](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetLogEvents.html) or [write](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html). This example breaks down one query into multiple queries if more than the maximum amount of records is returned from the query.
+
+The following components are used in this example:
+
+- [Amazon CloudWatch Logs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html) hosts the logs that are queried using the [Amazon CloudWatch Logs API](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/Welcome.html).
+
+## Supporting infrastructure
+
+An AWS CloudFormation [template](./resources/stack.yaml) exists in the [resources folder](./resources/). The template contains two resources: a CloudWatch log group, and a CloudWatch log stream that will be used as the default log group and stream for the queries made in the example. We recommend deploying a stack with this template.
+
+### Prerequisites
+
+If you need to, [install or update the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+### Deploy resources
+
+To deploy the stack using the template, run the following command:
+
+```
+aws cloudformation deploy --template-file stack.yaml --stack-name LargeQueryStack
+```
+
+### Destroy resources
+
+To destroy the stack, run the following command:
+
+```
+aws cloudformation delete-stack --stack-name LargeQueryStack
+```
+
+## Sample logs
+
+A lot of logs are needed to make a robust example. If you happen to have a log group with over 10,000 logs at the ready, great! If not, there are two resources that can help:
+
+### Prerequisites
+
+If you need to, [install or update the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
```

Comment:
Is this section needed here? It also appears earlier, and here it gets in between the lead-in to the resources and the list of resources.

Reply:
Good catch. It's an oversight.